templates/cloudflare-workers/package.json

```
@@ -26,7 +26,7 @@
     "@types/react-dom": "^18.2.7",
     "eslint": "^8.38.0",
     "typescript": "^5.1.6",
-    "wrangler": "^3.1.1"
+    "wrangler": "^3.8.0"
```

Whoops, this one needs to be fixed too

```suggestion
    "wrangler": "3.8.0"
```